types/fh-sync.d.ts

```diff
@@ -1,48 +1,97 @@
 declare module 'fh-sync' {
 
-  interface PendingWorkerBackoff {
-    strategy: string;
-    max: string;
-  }
-
-  interface AckWorkerBackoff {
-    strategy: string;
-    max: string;
-  }
-
-  interface SyncWorkerBackoff {
-    strategy: string;
-    max: number;
-  }
-
+  /**
+   * Options used to initialize Sync Server
+   */
   interface SyncGlobalOptions {
```

Comment:
@aidenkeating Setting the default values in the docs on interfaces also won't really work. They only work as parameters to functions (including constructors), not as properties on objects. As long as the type of the property isn't nullable, you'll know the value is always there, and whatever the default value is doesn't really matter much.